tests/helpers/nexus.py

```diff
@@ -0,0 +1,37 @@
+import temporalio.api
+import temporalio.api.common
```

Comment:
Some of these imports not needed I don't think

Reply:
Thanks, fixed.